x-pack/plugins/upgrade_assistant/public/application/components/tabs/checkup/deprecations/index_table.tsx

```diff
@@ -145,29 +152,41 @@ export class IndexDeprecationTable extends React.Component<
     // NOTE: this naive implementation assumes all indices in the table are
     // should show the reindex button. This should work for known use cases.
```

Comment:
Hi @alisonelizabeth, I think we could update the comment to include a short explanation for the index settings too, WDYT?

Reply:
Good catch! Updated.